src/App.ts

```diff
@@ -994,8 +958,8 @@ function singleTeamAuthorization(
           };
         });
 
-  return async () => {
-    return { botToken: authorization.botToken, ...(await identifiers) };
+  return async ({ isEnterpriseInstall }) => {
```

Comment:
probably just a brain fart, but where is this `isEnterpriseInstall` coming from here?

Reply:
Yeah higher order functions can be mind-bending for me too. `singleAuthorization()` returns a function, which as the type `Authorize`. So this expressing is of type `Authorize`. Which means the argument is of type `AuthorizeSourceData`. So this `isEnterpriseInstall` is the property of `AuthorizeSourceData`, [defined on line 94](https://github.com/aoberoi/bolt/blob/8266e6857f1602d9f98ff29e5b780cab4de2899e/src/App.ts#L94).

As for the actual value, its from the return value of `buildSource()`.